dev-python/sphinx-autoapi/sphinx-autoapi-1.8.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="A new approach to API documentation in Sphinx"
+HOMEPAGE="https://sphinx-autoapi.readthedocs.io/"
+SRC_URI="https://github.com/readthedocs/sphinx-autoapi/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+# pypi lacks docs/
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+RDEPEND="
+	dev-python/sphinx[${PYTHON_USEDEP}]
+	dev-python/astroid[${PYTHON_USEDEP}]
+"
+
+DOCS=( README.rst CHANGELOG.rst )
+
+# Test requires pypi download w/ internet: https://github.com/readthedocs/sphinx-autoapi/issues/329
+EPYTEST_DESELECT="tests/test_integration.py::TestExtensionErrors::test_extension_setup_errors[dotnetexample-override_conf2-AutoAPI"
```

Comment:
Seems like the closing bracket is missing.

Reply:
Nit: I'd prefer an array here () instead too

Reply:
> Seems like the closing bracket is missing.

Turns out there must be no closing bracket - i added it and it made it fail!

Reply:
The full test's pytest parameter is something stupid that I didn't want to drop the whole thing in:
```
tests/test_integration.py::TestExtensionErrors::test_extension_setup_errors[dotnetexample-override_conf2-AutoAPI of type `dotnet` requires following packages to be installed and included in extensions list: sphinxcontrib.dotnetdomain (available as "sphinxcontrib-dotnetdomain" on PyPI)]
```